bakery/src/scripts/check_feed.py

```diff
@@ -25,13 +25,17 @@ def main():
     # file in S3 bucket. If the book is not complete, check pending and retry
     # states to see whether or not it has errored or timed out too many times
     # to be queued again.
+
     for book in feed_data:
         # Check for loop exit condition
         if books_queued >= max_books_per_run:
             break
 
+        # FIX: For better clarity. Currently 'style' denotes 'slug' for content from Github.
+        book_identifier = book['collection_id'] if book['server'] else book['style']
```

Comment:
wouldn't this mean every entry in the `git-approved-books.json file` would have `server: null`. Additionally, I think it may be a bit confusing to have style imply slug, no? At least for right now, doesn't that prevent us from using the correct style and recipes on any book in this repo except for precalculus (since they all use the precalculus recipe? If you are checking `server` to see if it's null, why not just include a `slug` field and remove the `collection_id` field?

Reply:
You've made very good points, I created this with the assumption of one pipeline initially.  

Moving forward and using 2 abls (archive & git), I will remove `server` and `collection_id` add `slug` for the git abl.